docs/propagation.md

```diff
@@ -210,3 +217,58 @@ along F0, propagate `["c"]` along F1, and propagate nothing along F2.
 T0  | "a", **"b"** | **"c"**  | "f" |
 T1  | "a", "b"     | "c", "d" | "g" |
 T2  | **"a", "b"** | "c", "e" |     |
+
+### Data flow ops
+
+The above propagation step description applies to most ops. However, there are
+cases where a sharding rule is not appropriate. For those cases, Shardy defines
+_data flow _ ops.
```

Comment:
the underscores don't seem to do anything in markdown, also there is a space between flow the the underscore.

Reply:
I think maybe this was a typo, need to do `_data flow_` to get italics like _for example_

Reply:
Yes, typo! Sorry for that, will fix.